pkg/validation/internal/bundle.go

```diff
@@ -15,6 +17,9 @@ import (
 
 var BundleValidator interfaces.Validator = interfaces.ValidatorFunc(validateBundles)
 
+// max_bundle_sized_allowed defines the max size of the bundle
+const max_bundle_sized_allowed = 1048576
```

Comment:
@kevinrizza could you please help out just by double-checking that is the correct amount?

Reply:
c/c @perdasilva @njhale @anik120 could you please help us check if that amount is the right one and etc?